### DISCUSSION TITLE
Using the logged on user's profile for Chrome

### DISCUSSION THREAD

**Discussion by Cyb3r-Monk** [cmu-sei/GHOSTS#121] [Q&A] [answered]
Chrome uses a temporary profile like below:
`C:\Users\S50FB~1.CHI\AppData\Local\Temp\scoped_dir18316_422758565\Default`

Is it possible to specify the logged on user's profile so that history and other data remains?

**Comment by sei-dupdyke** [comment, marked as answer]
I made this an issue (#122), shouldn't be too difficult to implement. Thanks.
